client/my-sites/stats/activity-log/index.jsx

```diff
@@ -102,24 +102,8 @@ class ActivityLog extends Component {
 	}
 
 	getStartMoment() {
-		const { gmtOffset, moment, startDate, timezone } = this.props;
-
-		if ( timezone ) {
-			if ( ! startDate ) {
-				return moment().tz( timezone );
-			}
-
-			return moment.tz( startDate, timezone );
-		}
-
-		if ( null !== gmtOffset ) {
-			return moment
-				.utc( startDate )
-				.subtract( gmtOffset, 'hours' )
-				.utcOffset( gmtOffset );
-		}
-
-		return moment.utc( startDate );
+		const { gmtOffset, startDate, timezone } = this.props;
```

Comment:
This method is now just a wrapper around the pure utility function with questionably benefit. Perhaps it should be removed completely.